script/test/utils.sh

```diff
@@ -52,7 +52,14 @@ version=2
 
 [plugins."io.containerd.grpc.v1.cri"]
   drain_exec_sync_io_timeout = "10s"
+EOF
+  if command -v sestatus >/dev/null 2>&1; then
+    cat >>${config_file} <<EOF
+  enable_selinux = true
+EOF
+  fi
 
+  cat >>${config_file} <<EOF
```

Comment:
Can you move this to a separate commit?  It can still be in the same PR, but it's not really related to the Volume Source support changes.

Reply:
I have split out a new PR for this and will rebase this PR after the #10713 merge.